java/daikon/dcomp/DCInstrument.java

```diff
@@ -1812,26 +1856,70 @@ void add_exit(MethodGen mg, MethodInfo mi, int method_info_index) {
     }
   }
 
+  /**
+   * Search the interfaces of a class for an implementation of the target method. For each interface
```

Comment:
Both here in the documentation, and also in the method name, please focus on *what* the method does rather than *how* it does it (by searching).  A following paragraph in the Javadoc can give the algorithm.  That can be useful information, but not as the first thing, and not as something that is visible to clients at call sites.